include/libzfs.h

```diff
@@ -494,6 +494,7 @@ _LIBZFS_H zfs_type_t zfs_get_underlying_type(const zfs_handle_t *);
 _LIBZFS_H const char *zfs_get_name(const zfs_handle_t *);
 _LIBZFS_H zpool_handle_t *zfs_get_pool_handle(const zfs_handle_t *);
 _LIBZFS_H const char *zfs_get_pool_name(const zfs_handle_t *);
+_LIBZFS_H uint64_t zfs_get_txg_fast(const zfs_handle_t *);
```

Comment:
I think `zfs_get_createtxg_fast` would be more clear, or just `zfs_get_createtxg`. 

Grepping for `ZFS_PROP_CREATETXG` lands a bunch of hits in lib and cmd where we might switch to using this method, assuming `zhp->zfs_dmustats.dds_creation_txg` is always available in both full and simple handles now. Would you mind looking into that as well?

Reply:
Ditto. Maybe update zfs_prop_get() to return this if it's always available (this would also assume availability in normal dataset handles).

Reply:
Yes, you are absolutely right. I think we atleast need to replace ```zfs_prop_get_int(guid2hdl, ZFS_PROP_CREATETXG)``` with the faster version. I will update and will do thorough testing.